cas-server-support-jpa-ticket-registry/src/main/java/org/apereo/cas/config/JpaTicketRegistryConfiguration.java

```diff
@@ -47,6 +47,7 @@
     public String[] ticketPackagesToScan() {
         return new String[] {
                 "org.apereo.cas.ticket",
+                "org.apereo.cas.util.jpa",
```

Comment:
We need ZonedDateTimeConverter work to convert ZonedDateTime to java SQL type，we want the database column type is timestamp，not oid。